build.gradle.kts

```diff
@@ -132,14 +127,10 @@ subprojects {
         }
     }
 
-    tasks.register("pre") {
+    tasks.register("format") {
```

Comment:
renamed to match Creek - otherwise it'll drive me bonkers!  Hope that's OK.

Reply:
no worries